javascript.js

```diff
@@ -0,0 +1,20 @@
+// Set variavles
+var user = []; //inline coment
+
+/**
+ * Adds two numbers togeter
+ * 
+ * @param {number} first The First Number
+ * @returns {number}
+ */
+function add(first) {
+  /*
```

Comment:
Typo inside comment in token 'coment'. Possible corrections:comment (69),moment (39),cement (37)